kerberos/tests/integration_test.rs

```diff
@@ -0,0 +1,32 @@
+use kerberos::{
+    client::Client,
+    servers::{tcp_server::TcpServer, udp_server::UdpServer, Server},
+};
+
+#[tokio::test]
+async fn test_tcp_client_request_and_respond() {
```

Comment:
test name should be expressive. 
1. What do you test for in this function? 
2. Which behaviors are you expect from these components?